### ISSUE TITLE
Native folders are not created

### ISSUE THREAD

**Issue by chjan** [cjstehno/gradle-natives#24]
Minimizing the problem to one file only, like one of these alternatives, a dll or the dll packed in a jar,  a native folder is not created. Are you sure this plugin still works?

dependencies {
    //native library
    compile files('src/main/jnilibs/win_mdnsresponder64/mDNSResponder.dll')
    compile files('src/main/jnilibs/mDNSResponder.jar')
}

**Comment by cjstehno** [comment]
Nope. I have listed it as abandoned. I will try to take a look this week though.

Not sure it worked if the native library is outside of a jar (as your example) file though...that was never the intention.

What Gradle version?

**Comment by chjan** [comment]
latest gradle 4.2.1.

I you accept a suggestion, then it is a fact that the Java plugin of Gradle is not "up to standard" as it does not provide loading of JNI things (as Gradle does for Android). Gradle is not yet fully functional for pure Java, in other words. This is a bummer.

What is needed, is that your plugin is able to load native files into such an such folder in a simple and straight forward way. I don't understand the purpose of putting that type of files in a jar, but it is a working solution if it keeps your plugin more simple. Even better would be to allow direct, non-jared file load. As indicated above. This is what is still missing in the Gradle Java plugin (odd if you ask me!). This is what we most often do when we want to load native layers in Java.
Hope you refurbish this plugin along these lines, because it is needed.

**Comment by cjstehno** [comment]
I have officially abandoned this project. Feel free to submit pull requests to address issues if you want to. See the README for more information. Thanks.
